src/domain/landingPage/__tests__/LandingPage.test.tsx

```diff
@@ -2,18 +2,20 @@ import { act } from '@testing-library/react';
 import { axe } from 'jest-axe';
 import * as React from 'react';
 
+import translations from '../../../common/translation/i18n/fi.json';
 import {
   CollectionListDocument,
   LandingPagesDocument,
 } from '../../../generated/graphql';
+import * as useMobile from '../../../hooks/useMobile';
 import {
   fakeBanner,
   fakeCollections,
   fakeLandingPages,
   fakeLocalizedObject,
 } from '../../../util/mockDataUtils';
 import { render, screen } from '../../../util/testUtils';
-import LandingPage from '../LandingPage';
+import FrontPage from '../LandingPage';
```

Comment:
First I just created FrontPage component because I was using code from the old LandingPage component. I just changed the new component to LandingPage after deleting the old one. Not sure if LandingPage or FrontPage is better

Reply:
I think LandingPage is better name for import here, but later all the components could be renamed to `FrontPage*` if it's the "domain word" that every stakeholder understand. Anyway this is not a big deal, it's more like clean code rule that synonyms should be avoided.

Reply:
Yes of course. FrontPage was left there by accident. I fixed this yesterday.

Reply:
Oh I noticed that you already renamed it. Good